pkgs/haskell/cabal/cabal-install/registry.yaml

```diff
@@ -0,0 +1,15 @@
+packages:
+- name: haskell/cabal/cabal-install
+  repo_owner: haskell
+  repo_name: cabal
```

Comment:
Add `repo_owner` and `repo_name` for getting the version from GitHub Releases in `aqua g`.

Reply:
Ahh, great! I didn't know this was a way to get the version. I have another package I was considering that could use this.